components.json

```diff
@@ -33,25 +16,19 @@
             [ "make", "install" ]
         ]
     },
-    "nqp-parrot": {
-        "name":         "nqp-parrot",
+    "nqp-js": {
+        "name":         "nqp-js",
         "repo_url":     "git://github.com/perl6/nqp.git",
         "release_tags": "^ 20\\d\\d \\D \\d\\d [\\.\\d+]? $",
         "build_steps":  [
             [ "make", "realclean" ],
-            [ "git", "clean", "-dxf" ],
-            [ "rm", "-rf", "install", "parrot" ],
-            [ "git", "clone", "-l", "../../parrot/parrot.git" ],
-            [ "perl", "Configure.pl", "--backends=parrot", "--gen-parrot" ],
+            [ "rm", "-rf", "install", "MoarVM" ],
+            [ "git", "clone", "-l", "../../moarvm/moarvm.git", "MoarVM" ],
+            [ "perl", "Configure.pl", "--backends=moar,js", "--gen-moar" ],
             [ "make" ],
             [ "make", "install" ]
-        ]
```

Comment:
Should this be deleted?  It looks like it makes it mismatched.

Reply:
Whoops, good spot, fixed.